port/linux/omrvmem.c

```diff
@@ -1128,13 +1130,17 @@ reserve_memory_with_mmap(struct OMRPortLibrary *portLibrary, void *address, uint
 	int protectionFlags = PROT_NONE;
 	BOOLEAN useBackingFile = FALSE;
 	BOOLEAN useBackingSharedFile = FALSE;
+	BOOLEAN useBackingSharedTmpFile = FALSE;
 	BOOLEAN areHugePagesEnabled = FALSE;
 
 	Trc_PRT_vmem_reserve_entry(address, byteAmount);
 
 	if (0 != (mode & OMRPORT_VMEM_MEMORY_MODE_SHARE_FILE_OPEN)) {
 		flags |= MAP_SHARED;
 		useBackingSharedFile = TRUE;
+	} else if (0 != (mode & OMRPORT_VMEM_MEMORY_MODE_SHARE_TMP_FILE_OPEN)) {
```

Comment:
This should use `OMR_ARE_ANY_BITS_SET()`; ideally, other occurrences of this pattern should be fixed.